webapp/templates/public/scoreboard.html.twig

```diff
@@ -7,7 +7,7 @@
         {% if showExternalId(contest) %}
             {% set contestId = contest.externalid %}
         {% endif %}
-        {% set bannerImage = contestId | assetPath('contest') %}
+        {#% set bannerImage = contestId | assetPath('contest') %#}
```

Comment:
What happened when this was enabled?

Reply:
```

In DOMJudgeService.php line 1363:
                                                                                                                     
  App\Service\DOMJudgeService::assetPath(): Argument #1 ($name) must be of type string, null given, called in /domj  
  udge/webapp/var/cache/dev/twig/05/0524521e8604d78b0578c15c32703153.php on line 108
  ```

Reply:
Lets add a check if contestId is not null then?

Reply:
right yes done; that was easy in hindsight